### DISCUSSION TITLE
[Esame 27 Gennaio 2023] Esercizio 3

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#346] [Esami - Bandiera]
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/453439f0-6400-4e6c-b959-8aaf14b205a5)

**Comment by Elia-Belli** [comment]
**(a)** Utilizziamo l'eliminazione di Gauss per ridurre $A$ alla seguente matrice a scala:

$$S=\begin{bmatrix}1&0&2&3\\
0&1&1&7\\
0&0&1&10\\
0&0&0&0\end{bmatrix}$$

Osserviamo che i pivot si trovano sulle prime tre colonne di $S$, allora $Im(L_A)$ ha come base le prime tre colonne di $A$
Per la base di $Ker(L_A)$ risolviamo il seguente sistema con una variabile libera:

$$\begin{cases}x+2z+3w=0\\
y+z+7w=0\\
z+10w=0\\
w=w\end{cases}\to\begin{cases}x=17w\\
y=3w\\
z=-10w\\
w=w\end{cases}$$

Quindi la base di $Ker(L_A)$ è il vettore $(17,3,-10,1)$

**(b)** Conoscendo la base di $Im(L_A)$ (eq. parametriche) possiamo determinare le equazioni cartesiane imponendo la compatibilità sul seguente sistema:

$$\begin{cases}x+0y+2z=x_1\\
-3x+y-5z=x_2\\
x-2y+z=x_3\\
x+y+2z=x_4\end{cases}\to\begin{bmatrix}1&0&2\\
0&1&1\\
0&0&1\\
0&0&0\end{bmatrix}\begin{bmatrix}x_1\\
x_2+3x_1\\
x_3+2x_2+5x_1\\
x_1+x_2+x_3+x_4\end{bmatrix}\Rightarrow Im(L_A)=\\{\underline x\in \mathbb{R}^4|\begin{cases}x_1+x_2+x_3+x_4=0\end{cases}\\}$$

**Comment by CiottoloMaggico** [comment]
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/0291b8c4-9c6f-4a98-904b-8ae4a8c2c7ed)
